xla/tests/collective_ops_e2e_test.cc

```diff
@@ -659,6 +659,7 @@ class CollectiveOpsTestE2EWindowedNonWindowed : public CollectiveOpsTestE2E {
     auto opts = GetDebugOptionsForTest();
     opts.set_xla_gpu_threshold_for_windowed_einsum_mib(0);
     opts.set_xla_gpu_multi_streamed_windowed_einsum(true);
+    opts.set_xla_gpu_mlir_emitter_level(0);
```

Comment:
Why is this needed? Do you run into an issue with the MLIR emitters?

Reply:
i can remove this now. There was an issue with the mlir emitter level 1. We had a discussion on gchat, this has been changed to level 0 by default.